curated_transformers/tokenization/sentencepiece_adapters.py

```diff
@@ -47,3 +50,37 @@ def xlmr_adapter_forward(model: Model, X: List[Ragged], is_train: bool):
         )
 
     return X_xlmr, lambda dY: []
+
+
+def _camembert_update_to_fairseq(piece_id):
+    if piece_id == _SPP_UNK:
+        return _FAIRSEQ_UNK
+    else:
+        return piece_id + _CAMEMBERT_FAIRSEQ_OFFSET
+
+
+@lru_cache(maxsize=128)
+def _camembert_update_to_fairseq_vectorized(xp):
+    return xp.vectorize(_camembert_update_to_fairseq)
+
+
+def build_camembert_adapter() -> Model[List[Ragged], List[Ragged]]:
+    return Model(
+        "sentencepiece_encoder",
```

Comment:
This `Model` (and the one above) should probably be called `sentencepiece_adapter`.

Reply:
Named them after their constructors now.